### ISSUE TITLE
Adjustments to UI and readability

### ISSUE THREAD

**Issue by taniarascia** [halfer/php-tutorial-text#22]
Here are a few CSS adjustments for improved readability for your consideration.

#### Current:

```css
#content .sidebar {
    border: 1px dashed silver;
    margin: 0 4em;
    padding: 0 8px;
}

```

#### New:

```css
#content .sidebar {
    border: 1px dashed #247EDA;
    background: #e9f2fb;
    margin: 2em 0;
    padding: 1.5em;
}

#content .sidebar p {
    margin: 1.5em 0;
}
```

#### Current:

```css
#content p, #content ul, #content ol {
    font-size: 1.3em;
}

#content p {
    margin: 16px 0;
    line-height: 140%;
}
```

```css
#content code {
    font-size: 1.1em;
}
```

#### New:

```css
#content p, #content ul, #content ol {
    font-size: 1.1em;
    line-height: 1.6;
    font-weight: 400;
    margin: 1.5em 0;
}
```

```css
#content code {
    font-size: 1.1em;
    background: rgba(0, 0, 0, .08);
}

#content pre code {
    background: transparent
}
```

I would also probably make the alert note stand out more.

#### Current:

```css
#versions-note {
    min-width: 850px;
    max-width: 1000px;
    margin: 0 auto;
    font-family: "Open Sans",sans-serif;
    font-weight: 300;
    border: 1px dotted gray;
    color: gray;
}
````

#### New:

```css
#versions-note {
    border: 1px dotted #EEA09F;
    color: #B02323;
    background: #fbeceb;
    line-height: 1.6;
    font-weight: bold;
}

#versions-note #versions-note-inner {
    padding: 1.5em;
    font-size: 1em;
}
```

```css
#content {
   color: #333333;
}
```

[Here's an example image](http://imgur.com/a/3V3KU).

I would also consider removing `margin-top: 0` from `h1` and `#contents-menu h2`, as well as making headings bold.

**Comment by halfer** [comment]
That's great, thanks! I've been meaning to set up a build system for the tutorial ages, as it is a pain to build manually, and my initial build attempts using Ansible did not quite get finished. So, this has given me the impetus today to get it done in Docker, now 90% done.

I will push that into live probably tomorrow, and then I can add your suggested changes easily.

**Comment by taniarascia** [comment]
Awesome! Glad you found it useful. Looking forward to seeing the update! Feel free to ask if anything doesn't make sense.

**Comment by halfer** [comment]
That's great, thanks very much! I've implemented all of those on my staging server, with minor deviations:

* The combination of a generous `<p>` vert margin and a large `<blockquote>` vert padding meant that pullquotes had a bit too much top and bottom inner space. I've reduced that a bit.
* Inline `<code>` items now have a `class="inline"` so as to help differentiate from code blocks. I put in a little bit of inner padding and rounded the corners slightly.
* I've not added the versions warning styles, as I was thinking of hiding that anyway: it is most unlikely that someone is still working on an old version from 2014! :smile_cat: That said it may be useful again if I were to make text changes in the future.

I'll send you the staging server details privately, for your feedback. See how it looks, and let me know if there's anything you'd change? I might tweak the line `<code class="inline">` formatting device so it is similar to `<span class="filename">`, perhaps differentiating them slightly on colour, but it is a minor thing.

**Comment by taniarascia** [comment]
It's looking really good! Much easier for me to read. I would still personally add bold to the headlines to make them stand out/not blend in with the main copy, and top margins, but it's definitely looking good.  

Considering how wide your code blocks are (since they're side by side) I have to do a lot of horizontal scroll to look through them. Ideally, the way to fix this situation would be to remove the min and max width from your main `.content` div, put a max-width and change the margins to `margin: 1.5em auto` on all your regular elements (quotes, paragraphs, lists) which would [look like this](http://imgur.com/QeyOGS5), but of course that's up to you. It's helpful for *me* because it prevents horizontal scroll on code blocks, and stops me from having to move my eyes all across the screen for very wide blocks of paragraph text.

Just some comments.

**Comment by halfer** [comment]
OK, thanks - I have promoted the staging version to live now.

Regarding bolder headlines, I'd be happy to try that. I'm ironing out some Docker hiccups presently, but will give that a go.

Thanks for the suggestion above the width of the diffs, I see what you mean. I think I'd want to preserve the same width on initial rendering, as I think that's a neater presentation. However, I've suggested a couple of ideas on fixing that in #23.
